File: Python/Daily_Code_Challenge/250928-get-headings.py
def get_headings(csv):

    headings = [heading.strip() for heading in csv.split(",")]
    print(headings)

    return headings

get_headings("username , email , signup date ")

'''
CSV Header Parser

Given the first line of a comma-separated values (CSV) file, return an array containing the headings.

    The first line of a CSV file contains headings separated by commas.
    Remove any leading or trailing whitespace from each heading.

Tests

    Passed: 1. get_headings("name,age,city") should return ["name", "age", "city"].
    Passed: 2. get_headings("first name,last name,phone") should return ["first name", "last name", "phone"].
    Passed: 3. get_headings("username , email , signup date ") should return ["username", "email", "signup date"].
'''

File: JavaScript/DailyCodeChallenge/250928_getHeadings.js
function getHeadings(csv) {

  const splitHead =csv.split(",")

  console.log(splitHead)

  const answer =  splitHead.map(item => {
    return `${item.trim()}`
  })
  console.log(answer)
  return answer;
}

getHeadings("username , email , signup date ")

/*
CSV Header Parser

Given the first line of a comma-separated values (CSV) file, return an array containing the headings.

    The first line of a CSV file contains headings separated by commas.
    Remove any leading or trailing whitespace from each heading.

Tests

    Passed: 1. getHeadings("name,age,city") should return ["name", "age", "city"].
    Passed: 2. getHeadings("first name,last name,phone") should return ["first name", "last name", "phone"].
    Passed: 3. getHeadings("username , email , signup date ") should return ["username", "email", "signup date"].
    */